app/assets/javascripts/discourse/app/components/user-menu/items-list.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if this.loading}}
+  <div class="spinner-container">
+    <div class="spinner"></div>
+  </div>
```

Comment:
Are we able to use the `<ConditionalLoadingSpinner>` component here instead?

Reply:
Same as https://github.com/discourse/discourse/pull/17492#discussion_r922717174, what are your thoughts?